app/views/admin/locations/_form.html.erb

```diff
@@ -1,4 +1,16 @@
-<%= form_for location, url: admin_location_path do |f| %>
+<%= form_for location, url: admin_location_path, method: 'put' do |f| %>
+  <% if location.errors.any? %>
+    <div class="alert alert-danger t-errors" role="alert">
+      <h2 class="alert__heading">There's a problem</h2>
```

Comment:
We would typically link to the fields that have errors on them, and display inline warnings next to each of them too. See: https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/app/views/calculators/_error_summary.html.erb and https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/app/views/calculators/_form_input_currency.html.erb